### PR TITLE
TW-747 Add 'Main info' section to 'Savings' page

### PR DESCRIPTION
**Pull request by keshan3262**
This PR should be merged after #855

### CODE REVIEW

**Review by svjatoslavtt — APPROVED** [review]

File: src/screens/farming/constants.ts [hunk 1]
```diff
@@ -1,3 +1,2 @@
 export const DEFAULT_AMOUNT = 0;
-export const DEFAULT_DECIMALS = 2;
 export const PENNY = 0.01;
```
Comment (Boltman92) [inline src/screens/farming/constants.ts:2]: we have the same constants  in `src/components/earn-opportunities-main-info/constants.ts`

maybe better to reuse one of it?

**Review by Boltman92 — APPROVED** [review]

**Review by Boltman92 — APPROVED** [review]

**Review by Boltman92 — APPROVED** [review]

**Review by svjatoslavtt — APPROVED** [review]

**Review by herkoss — APPROVED** [review]
GJ!

**Review by svjatoslavtt — APPROVED** [review]

**Review by vladvn1905 — APPROVED** [review]
ios done 1.15_791

**Review by IrynaKhyzhynska — APPROVED** [review]
android done 1.15_791

**Pull request merged**